contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -5499,6 +5500,7 @@ bbf_ExecDropStmt(DropStmt *stmt)
 
 		foreach(cell, stmt->objects)
 		{
+			relation = NULL;
```

Comment:
let us add tests for TYPE as well. Also, add test for schemas (multiple schema as well even if it gives error)

Reply:
done